packages/js/src/classic-editor/watcher.js

```diff
@@ -377,7 +377,7 @@ const syncStoreToPost = () => {
 	createDomSync( selectors.selectSeoTitle, { domGet: dom.getPostSeoTitle, domSet: dom.setPostSeoTitle }, "seoTitle" );
 	createDomSync( selectors.selectMetaDescription, { domGet: dom.getPostMetaDescription, domSet: dom.setPostMetaDescription }, "metaDescription" );
 	createDomSync( selectors.selectKeyphrase, { domGet: dom.getPostFocusKeyphrase, domSet: dom.setPostFocusKeyphrase }, "focusKeyphrase" );
-	createDomSync( selectors.shouldApplyCornerstoneAnalysis, { domGet: dom.getPostIsCornerstone, domSet: dom.setPostIsCornerstone }, "isCornerstone" );
+	createDomSync( selectors.selectShouldApplyCornerstoneAnalysis, { domGet: dom.getPostIsCornerstone, domSet: dom.setPostIsCornerstone }, "isCornerstone" );
```

Comment:
This solves the SEO and readability analysis not being updated when toggling the cornerstone toggle, since the selector was renamed from `shouldApplyCornerstoneAnalysis` to `selectShouldApplyCornerstoneAnalysis` in the process of creating [this PR](https://github.com/Yoast/wordpress-seo/pull/18510). This rename was forgotten.